OpenFeature/src/main/java/dev/openfeature/sdk/FeatureProvider.kt

```diff
@@ -5,10 +5,14 @@ interface FeatureProvider {
     val metadata: ProviderMetadata
 
     // Called by OpenFeatureAPI whenever the new Provider is registered
-    suspend fun initialize(initialContext: EvaluationContext?)
+    fun initialize(initialContext: EvaluationContext?)
```

Comment:
I wonder if we should keep "suspend" in the interface towards the Provider, and only remove "suspend" from the OpenFeature user-facing API `setProvider`

Reply:
i think it makes sense that now we have the shutdown function we move the responsibility of having scopes and suspending in the provider side. but we can further discuss this.